.github/workflows/publish_releasenotes_outside.yml

```diff
@@ -0,0 +1,26 @@
+#############################################################################
+# GitHub Action to Publish Release Notes Outside.
+#
+#############################################################################
+name: "Publish Release notes Outside"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New Version'
+        required: true
+jobs:
+  publish:
+    name: Publish releasenotes outside ${{ github.event.inputs.version }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Download xmlstarlet
+        run: |
+          sudo apt-get install xmlstarlet
+      - name: Run Shell Script
+        run: |
+          ./.ci/publish-releasenotes-outside.sh ${{ github.event.inputs.version }}
```

Comment:
outside --> ghpage for yml and shell file

Reply:
change name of action also.

Reply:
Changes Done